packages/neon_framework/packages/account_repository/lib/src/utils/authentication_client.dart

```diff
@@ -0,0 +1,44 @@
+// ignore_for_file: prefer_prefixed_nextcloud_import
+
+import 'package:meta/meta.dart';
+import 'package:nextcloud/core.dart' as nc_core;
+import 'package:nextcloud/nextcloud.dart';
+import 'package:nextcloud/provisioning_api.dart' as nc_provisioning_api;
```

Comment:
```suggestion
import 'package:nextcloud/core.dart' as core;
import 'package:nextcloud/nextcloud.dart';
import 'package:nextcloud/provisioning_api.dart' as provisioning_api;
```
(CI should also tell you this with the new rule)

Reply:
Not if you ignore the rule :P
In this case I ignored the rule so the client being called is still `client.xxx.core` instead of `client.xxx.coreClient`.
Having the field named core would conflict with the import prefix.

This is also needed for our `find_untested_apis.sh` script to funciton.

Reply:
You can prefix it with `$` which is the alternative for these cases (also happened in notes and notifications).